Client/sdk/game/CAutomobile.h

```diff
@@ -55,8 +57,9 @@ class CAutomobile : public virtual CVehicle
     virtual void       SetRandomDamage(bool bUnknown) = 0;
     virtual void       SetTaxiLight(bool bState) = 0;
     virtual void       SetTotalDamage(bool bUnknown) = 0;
-    virtual void       SetTowLink(CVehicle* pVehicle, bool bRemoveAdd) = 0;
+    //virtual void       SetTowLink(CVehicle* pVehicle, bool bRemoveAdd) = 0;
```

Comment:
I am confused as to why this is commented out, and someone in the future will look at this and feel the same way. Either remove the line or add a code comment explaining why it's commented.

Reply:
I removed the comment in the latest commit. CAutomobile class was never used in the client project, so this function was never implemenented in the derived class. After I added support for CAutomobile, it was throwing compile-time errors. I was going to implement it but forgot, removed it for now as it's not used.